cmd/dreamboat/main.go

```diff
@@ -438,6 +459,22 @@ func run() cli.ActionFunc {
 			return err
 		}
 
+		var exporter relay.DataExporter
+		if c.Bool("data-export") {
```

Comment:
A presence of non-zero data-export-workers (that should be 0 by default) would be more than enough to enable that